django/contrib/admin/static/admin/css/sidebar.css

```diff
@@ -51,24 +59,36 @@
     margin-right: -276px;
 }
 
-.toggle-nav-sidebar::before {
+#toggle-nav-sidebar::before,
+.main.changelist-filter-expanded #toggle-filter-sidebar::before {
+    /* arrow right */
     content: '\00BB';
 }
 
-.main.shifted .toggle-nav-sidebar::before {
+#toggle-filter-sidebar::before,
+.main.nav-sidebar-expanded #toggle-nav-sidebar::before {
+    /* arrow left */
     content: '\00AB';
 }
 
+.main > #changelist-filter {
+    display: none;
```

Comment:
using `display` instead of `visibility` (as `#nav-sidebar` does) because it doesnt need additional justifications via in `flex` and `margin` (as `#nav-sidebar` does)